### PR TITLE
'pip3 install openshift' -> 'dnf install python3-openshift'

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,9 @@
-FROM registry.fedoraproject.org/fedora:29
+FROM registry.fedoraproject.org/fedora:30
```
Comment (TomasTomecek) [inline Dockerfile:1]: pretty bold

**Review by TomasTomecek — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (jpopelka) [reply Dockerfile:1]: what could possibly go wrong :-)

Comment (TomasTomecek) [reply Dockerfile:1]: right?!

**Pull request merged**